src/go/rpk/pkg/config/format.go

```diff
@@ -0,0 +1,38 @@
+package config
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"gopkg.in/yaml.v3"
+)
+
+type Formatter struct {
```

Comment:
icwym
+1 to keeping here for now
I think we can change the name a bit though, since this in usage, this reads as a config formatter -- this is any output formatter. Maybe `OutFormatter`?